src/FINDOLOGIC/Export/Data/DateAdded.php

```diff
@@ -17,16 +16,16 @@ public function setValue($value, $usergroup = '')
         throw new \BadMethodCallException('Assign DateAdded values by passing a \DateTime to setDateValue()');
     }
 
-    public function setDateValue(DateTime $value, $usergroup = '')
+    public function setDateValue(\DateTime $value, $usergroup = '')
```

Comment:
I actually prefer `use`ing everything, even standard library top-level classes. One backslash less in the code is one little bit less of clutter.

On the other hand, if we want to make this a rule to not `use` top-level standard library classes, we should probably teach the linter to make sure of that. Otherwise, it's just another optical detail that distracts from possible functional issues while reading the code.

Reply:
I followed your advise